docs/guides/fabric_synchronization_guide.md

```diff
@@ -82,11 +85,17 @@ fabricsync enable-auto-sync 1
 Pair the Fabric-Source bridge to Fabric-Sync with node ID 1:
 
 ```
-fabricsync add-bridge 1 <fabric-sink-ip>
+fabricsync add-bridge 1 <fabric-source-bridge-ip>
 ```
 
 ### Pair Light Example to Fabric-Source
 
+Since Fabric-Bridge also functions as a Matter server, running it alongside the
+Light Example app on the same machine would cause conflicts. Therefore, you need
+to run the Matter Light Example app on a separate physical machine from the one
+hosting Fabric-Admin and Fabric-Bridge. You can then commission the Matter Light
+Example app using Fabric-Admin on the source side.
```

Comment:
This can be worked around by using a different port and also not reusing the same default KVS path for multiple apps on the same machine. I have been running the following for instance:

```sh
# Bridge App with non-default discriminator/passcode and its own KVS path
./out/linux-x64-fabric-bridge-no-ble-clang-rpc/fabric-bridge-app --discriminator 3842 --passcode 20202022 --KVS /tmp/chip_kvs_fabric_bridge_app

# Lock App with different discriminator/passcode, ports and KVS
./out/linux-x64-lock-clang/chip-lock-app --discriminator 3846 --passcode 20202026 --secured-device-port 5546 --unsecured-commissioner-port 5556 --KVS /tmp/chip_kvs_lock_app

# Light App with yet another set of different discriminator/passcode, ports and KVS
./out/linux-x64-light-clang/chip-lighting-app --discriminator 3843 --passcode 20202023 --secured-device-port 5543 --unsecured-commissioner-port 5553 --KVS /tmp/chip_kvs_lighting_app
```

Reply:
Added, thanks for the tip